freedomofpress/securedrop
# Add label images

## Status

Ready for review.

## Description of Changes

Fixes #1141 .

Changes proposed in this pull request:
Inclusion of printable labels and a note on what to do with them.

## Testing

How should the reviewer test this PR?
make docs, preview locally in dev environment. The Sphinx download tag "should work" in other environments, but these are famous last words.

## Deployment
Assuming image assets are uploaded to a real local filesystem and not into some cloud thing with special replication processes and CDN URLs, git copying the image assets into the directory should just work.

## Checklist

### If you made changes to the app code:


### If you made changes to the system configuration:


^ I didn't think the docs were in the scope of unit/integration tests, but I'm happy to run them anyway if required!
